pkg/convert/tf.go

```diff
@@ -952,6 +952,12 @@ func convertObjectConsExpr(state *convertState, inBlock bool, scopes *scopes,
 	fullyQualifiedPath string, expr *hclsyntax.ObjectConsExpr,
 ) hclwrite.Tokens {
 	items := []hclwrite.ObjectAttrTokens{}
+
+	// Sort the items by line number so that we can deterministically order them in the output
```

Comment:
I don't think these sorts are actually needed, pretty sure Items is already going to be in declaration order.

Reply:
Again I'm pretty sure Items is already sorted, if so this is just extra code for no benefit.